curvefs/src/client/fuse_s3_client.cpp

```diff
@@ -304,7 +304,8 @@ void FuseS3Client::WarmUpAllObjs(
             context->cb = cb;
             s3Adaptor_->GetS3Client()->DownloadAsync(context);
         }
-        cond.Wait();
+        if (pendingReq.load())
```

Comment:
It seems useless, please explain why？

Reply:
if all objs has cached, then the function will block